src/Vm.sol

```diff
@@ -172,4 +172,33 @@ interface Vm {
     function deriveKey(string calldata, uint32) external returns (uint256);
     // Derive a private key from a provided mnenomic string (or mnenomic file path) at the derivation path {path}{index}
     function deriveKey(string calldata, string calldata, uint32) external returns (uint256);
+        // Returns the path to the project root, () => (path)
+    function projectRoot() external returns (string memory);
```

Comment:
The CI is failing because this PR adds `projectRoot`, but we already have it in `Vm`.

```suggestion
```

Reply:
But locally I only have a single `projectRoot` in my `vm.sol` ?

Reply:
We merged it in https://github.com/foundry-rs/forge-std/pull/158, so your `Vm` became outdated and Git resolved your `projectRoot` as a new addition to `Vm` here.